lib/msal-browser/src/interaction_client/SilentCacheClient.ts

```diff
@@ -78,13 +79,13 @@ export class SilentCacheClient extends StandardInteractionClient {
     }
 
     /**
-     * Currently Unsupported
+     * API to silenty clear the browser cache.
+     * @param logoutRequest
      */
-    logout(): Promise<void> {
-        // Synchronous so we must reject
-        return Promise.reject(
-            BrowserAuthError.createSilentLogoutUnsupportedError()
-        );
+    logout(logoutRequest?: EndSessionRequest): Promise<void> {
```

Comment:
Same here, I do not see validation of the optional param or the fallback code when `logout` is called without the `request`.

Reply:
What fallback are you expecting here? The only relevant piece for clearing cache is the account. If account was passed we will clear artifacts for just that account - if it was not we will clear artifacts for all accounts. This logic exists in clearCacheOnLogout

Reply:
I don't think we need fallback code here since we are checking for account in the `clearCacheOnLogout` method. It is not a required parameter, and the default case is being handled down the line.

Reply:
Ah okay. So `EndRequest -> Account` and we check for account. Makes sense. Next q I had is already covered before, can we thin down the `request` in that case.